### PR TITLE
Operating NFS service,  manual NFS export

### LINKED ISSUE #765: [doc] 15.4  Create an NFS Export
https://documentation.suse.com/ses/7/single-html/ses-admin/#ceph-nfsganesha-create-export

We can expand it and explain with an example on how each of the three config files should look like: bootstrap-config, common-config and export-config (Similar to confluence doc created by @mgfritch). Without using dashboard, deploying nfs-ganesha would require end user to modify lot of config files by hand. An example would make it easier to follow. 

As we will be providing only a basic export file example, we should ask users to refer to man pages for more options.

### LINKED ISSUE #767: [doc] 15.1  Starting or Restarting NFS Ganesha
### Source Document
15.1  Starting or Restarting NFS Ganesha:

https://documentation.suse.com/ses/7/single-html/ses-admin/#ceph-nfsganesha-services

We should specify that nfs-ganesha service would start but it won't do anything useful unless user has created export-config and common-config. Only after export-config and common-config are created, user can export cephfs over nfs-ganesha.

A easier way would be to first explain how to setup the configuration file and then place this section: Starting or Restarting NFS Ganesha. Then it would be clear that user is expected to have right configuration in place to get a working NFS-Ganesha.

### PR DESCRIPTION
**Pull request by tbazant**
This update improves the understand of what the Ganesha service does not and how to manually create an NFS export

Closes #767  
Closes #765

### CODE REVIEW

File: xml/admin_nfsganesha.xml [hunk 1]
```diff
@@ -101,6 +101,15 @@
  <sect1 xml:id="ceph-nfsganesha-services">
   <title>Starting or Restarting &ganesha;</title>
 
+  <important>
+   <para>
+    Operating the &ganesha; service does not export a &cephfs; file system. To
```
Comment (asettle) [inline xml/admin_nfsganesha.xml:106]:
```suggestion
    The operation the &ganesha; service does not automatically export a &cephfs; file system. To

Comment (asettle) [reply xml/admin_nfsganesha.xml:106]: Is that what you mean?

File: xml/admin_nfsganesha.xml [hunk 2]
```diff
@@ -139,17 +148,49 @@
...
-   Execute the following to create an NFS export.
+   You can create an NFS export either in the &dashboard;, or manually on the
+   command line. To create the export by using the &dashboard;, refer to
+   <xref linkend="dash-webui-nfs"/>, more specifically to
+   <xref linkend="dash-webui-nfs-create"/>.
   </para>
-
-  <note>
+  <para>
+   To create an NFS export manually, you first need to create a configuration
```
Comment (asettle) [inline xml/admin_nfsganesha.xml:157]:
```suggestion
   To create an NFS export manually, create a configuration

**Review by asettle — COMMENTED** [review]
Just comments, the first sentence is a bit odd to me?

File: xml/admin_nfsganesha.xml (hunk 1 above)
Comment (tbazant) [inline xml/admin_nfsganesha.xml:106]:
```suggestion
    Starting the &ganesha; service does not automatically export a &cephfs; file system. To
```

File: xml/admin_nfsganesha.xml [hunk 3]
```diff
@@ -101,6 +101,15 @@
  <sect1 xml:id="ceph-nfsganesha-services">
   <title>Starting or Restarting &ganesha;</title>
 
+  <important>
+   <para>
+    Starting the &ganesha; service does not automatically export a &cephfs; file system. To
+    export a &cephfs; file system, you first need to create an export
```
Comment (tbazant) [inline xml/admin_nfsganesha.xml:107]:
```suggestion
    export a &cephfs; file system, create an export
```

**Review by tbazant — COMMENTED** [review]
minor update

File: xml/admin_nfsganesha.xml [hunk 4]
```diff
@@ -139,17 +148,49 @@
...
+        user_id = "admin";
+        filesystem = "a";
+        secret_access_key = "<replaceable>SECRET_ACCESS_KEY</replaceable>";
+    }
+}
+</screen>
    <para>
-    The FSAL block should be modified to include the desired cephx user ID and
-    secret access key.
+    After the export file is created and saved on the file system, run the
```
Comment (lproven) [inline xml/admin_nfsganesha.xml:179]:
```suggestion
    After you have created and saved the configuration file for the new export,
    run the
```

Comment (lproven) [reply xml/admin_nfsganesha.xml:179]: Reword to avoid passive voice.

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by lproven — APPROVED** [review]

**Review by lproven — CHANGES_REQUESTED** [review]
I see 2 or more batches of changes. I didn't realise that requesting changes in 1, then approving another change, would approve them all. Sorry.

**Review by lproven — APPROVED** [review]

File: xml/admin_nfsganesha.xml [hunk 5]
```diff
@@ -139,17 +148,50 @@
...
+   You can create an NFS export either in the &dashboard;, or manually on the
+   command line. To create the export by using the &dashboard;, refer to
+   <xref linkend="dash-webui-nfs"/>, more specifically to
+   <xref linkend="dash-webui-nfs-create"/>.
   </para>
-
-  <note>
+  <para>
+   To create an NFS export manually, create a configuration
+   file for the export. For example, the file <filename>/tmp/export-1</filename>
```
Comment (lproven) [inline xml/admin_nfsganesha.xml:158]:
```suggestion
   file for the export. For example, a file <filename>/tmp/export-1</filename>
```

File: xml/admin_nfsganesha.xml [hunk 6]
```diff
@@ -139,17 +148,50 @@
...
+        secret_access_key = "<replaceable>SECRET_ACCESS_KEY</replaceable>";
+    }
+}
+</screen>
    <para>
-    The FSAL block should be modified to include the desired cephx user ID and
-    secret access key.
+    After you have created and saved the configuration file for the new export,
+    run the
+    following command to create the export:
```
Comment (lproven) [inline xml/admin_nfsganesha.xml:181]:
```suggestion
    After you have created and saved the configuration file for the new export,
    run the following command to create the export:
```

File: xml/admin_nfsganesha.xml [hunk 7]
```diff
@@ -139,17 +148,50 @@
...
    </para>
-  </note>
+<screen>rados --pool <replaceable>POOL_NAME</replaceable> --namespace <replaceable>NAMESPACE_NAME</replaceable> put <replaceable>EXPORT_NAME</replaceable> <replaceable>EXPORT_CONFIG_FILE</replaceable></screen>
+   <para>
+    For example:
+   </para>
+<screen>&prompt.cephuser;rados --pool example_pool --namespace example_namespace put export-1 /tmp/export-1</screen>
+   <note>
+    <para>
+     The FSAL block should be modified to include the desired cephx user ID and
```
Comment (lproven) [inline xml/admin_nfsganesha.xml:190]:
```suggestion
     The FSAL block should be modified to include the desired &cephx; user ID and
```

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by lproven — APPROVED** [review]

**Review by callithea — APPROVED** [review]
Lgtm

**Pull request merged**